### PR TITLE
Change references from dcaf soft pledge to fund pledge

### PR DESCRIPTION
**Pull request by colinmcglynn**
I rule and have completed some work on Case Manager that's ready for review!

I removed soft pledge and replace with #{FUND} pledge
I also replaced DCAF with the FUND constant where I could.  I figured we would want to continue to remove as many references in the code to DCAF as possible so that it will be easier to port over to other funds.  However, it is a bit out of scope, so let me know if you want me to revert any of it or move it to a new PR.

This pull request makes the following changes:
* replaces all instances of soft pledge with #{FUND} pledge
* replaces some instances of DCAF with #{FUND}
* removed DCAF from the names of the pledge limit environmental variables (NOTE: these will need to be updated in Heroku as well)

It relates to the following issue #s: 
* Fixes https://github.com/DCAFEngineering/dcaf_case_management/issues/1125

### CODE REVIEW

File: app/helpers/patients_helper.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@ def income_options
 
   def referred_by_options
     [nil, 'Clinic', 'Crime victim advocacy center',
-     'DCAF website or social media',
+     "#{FUND} website or social media",
```
Comment (colinxfleming) [inline app/helpers/patients_helper.rb]: note to us that this'll require a migration

File: app/helpers/patients_helper.rb [hunk 2]
```diff
@@ -67,10 +67,10 @@ def disable_continue?(patient)
     patient.pledge_info_present? ? 'disabled="disabled"' : ''
   end
 
-  def dcaf_pledge_limit_help_text
-    first_tri = ENV['DCAF_PLEDGE_LIMIT_FIRST_TRI'] || 100
-    second_tri = ENV['DCAF_PLEDGE_LIMIT_SECOND_TRI'] || 300
-    later_care = ENV['DCAF_PLEDGE_LIMIT_LATER_CARE'] || 600
+  def pledge_limit_help_text
+    first_tri = ENV['PLEDGE_LIMIT_FIRST_TRI'] || 100
```
Comment (colinxfleming) [inline app/helpers/patients_helper.rb]: should swap these env vars over to `DARIA_PLEDGE_LIMIT_xxx` in keeping with the naming scheme

File: test/models/patient_test.rb [hunk 3]
```diff
@@ -345,11 +345,11 @@ class PatientTest < ActiveSupport::TestCase
       assert @patient.valid?
     end
 
-    it 'should not validate pledge_sent if the DCAF soft pledge field is blank' do
+    it "should not validate pledge_sent if the #{FUND} pledge field is blank" do
       @patient.fund_pledge = nil
       @patient.pledge_sent = true
       refute @patient.valid?
-      assert_equal ['DCAF soft pledge field cannot be blank'], @patient.errors.messages[:pledge_sent]
+      assert_equal ["#{FUND} pledge field cannot be blank"], @patient.errors.messages[:pledge_sent]
```
Comment (colinxfleming) [inline test/models/patient_test.rb]: This defaults to DCAF so it's okay to leave it as DCAF

File: test/integration/update_patient_info_test.rb [hunk 4]
```diff
@@ -60,7 +60,7 @@ class UpdatePatientInfoTest < ActionDispatch::IntegrationTest
       fill_in 'Abortion cost', with: '300'
       fill_in 'Patient contribution', with: '200'
       fill_in 'National Abortion Federation pledge', with: '50'
-      fill_in 'DCAF pledge', with: '25'
+      fill_in "#{FUND} pledge", with: '25'
```
Comment (colinxfleming) [inline test/integration/update_patient_info_test.rb]: Can leave this as DCAF as well

File: test/models/patient_test.rb [hunk 5]
```diff
@@ -372,15 +372,15 @@ class PatientTest < ActiveSupport::TestCase
       @patient.appointment_date = nil
       @patient.pledge_sent = true
       refute @patient.valid?
-      assert_equal ['DCAF soft pledge field cannot be blank', 'Clinic name cannot be blank', 'Appointment date cannot be blank'],
+      assert_equal ["#{FUND} pledge field cannot be blank", 'Clinic name cannot be blank', 'Appointment date cannot be blank'],
```
Comment (colinxfleming) [inline test/models/patient_test.rb]: Can revert this to DCAF

File: test/models/patient_test.rb [hunk 6]
```diff
@@ -372,15 +372,15 @@ class PatientTest < ActiveSupport::TestCase
...
       @patient.errors.messages[:pledge_sent]
     end
 
     it 'should have convenience methods to render in view, just in case' do
       refute @patient.pledge_info_present?
       @patient.fund_pledge = nil
       assert @patient.pledge_info_present?
-      assert_equal ['DCAF soft pledge field cannot be blank'], @patient.pledge_info_errors
+      assert_equal ["#{FUND} pledge field cannot be blank"], @patient.pledge_info_errors
```
Comment (colinxfleming) [inline test/models/patient_test.rb]: Can revert this to DCAF

File: test/integration/data_entry_test.rb [hunk 7]
```diff
@@ -27,7 +27,7 @@ class DataEntryTest < ActionDispatch::IntegrationTest
       fill_in 'City', with: 'Washington'
       fill_in 'State', with: 'DC'
       fill_in 'County', with: 'Wash'
-      fill_in 'DCAF soft pledge', with: '100'
+      fill_in "#{FUND} pledge", with: '100'
```
Comment (colinxfleming) [inline test/integration/data_entry_test.rb]: Can revert this to DCAF

**Comment by colinmcglynn** [comment]
@colinxfleming your changes are in and tests are passing!

**Pull request merged**